### ISSUE TITLE
prioritize genomes found in previous samples when running genome grist

### ISSUE THREAD

**Issue by clb21565** [dib-lab/genome-grist#291]
not sure if this is baked in already, but when running many samples from the same experiment, it would be nice if the same species/strain genome representative was prioritized for each sample in the set before picking a different closely related genome

**Comment by ctb** [comment]
yeah, we struggle with this a bit in general. One thing you can do is put a database of the previously found genomes first on the sourmash_databases list - that might help. Please leave this open and I will see if I can test it and give you more guidance.

**Comment by clb21565** [comment]
thanks! currently I am just handling it on the back end by clustering the genomes of interest. it works OK.
